lib/drivers/insuletDriver.js

```diff
@@ -19,17 +19,37 @@ var _ = require('lodash');
 var struct = require('./../struct.js')();
 var sundial = require('sundial');
 
+var annotate = require('../eventAnnotations');
+var common = require('../insulet/common');
+var logic = require('../insulet/objectBuildingLogic');
+var insuletSimulatorMaker = require('../insulet/insuletSimulator');
+
+var isBrowser = typeof window !== 'undefined';
+var debug = isBrowser ? require('../bows')('InsuletDriver') : console.log;
+
 module.exports = function (config) {
   var cfg = _.clone(config);
   var buf;
   var bytes;
 
+  var BG_UNITS = 'mg/dL';
+
   // all insulin unit readings are in .01 unit increments, so we divide by 100.0 to get units
   // (multiplying by 0.01 tends to cause floating point issues)
   var toUnits = function (x) {
     return x / 100.0;
   };
 
+  // Insulet reports percentage temp basals as +/- integers [-100, 100+]
+  // i.e., -50 for 50% of the scheduled basal rate
+  var convertPercentage = function(x) {
```

Comment:
Could we maybe get a better name for this? convertPercentageToBasalFactor, perhaps? Or something? Esp since it returns null sometimes, when it's called it would help to know more about what it does.